wallet/src/de/schildbach/wallet/rates/ExchangeRate.java

```diff
@@ -68,6 +68,9 @@ public String getCurrencyName() {
         if (currencyCode.equalsIgnoreCase("VES")) {
             return "Venezuelan Bolívar";
         }
+        if (currencyCode.equalsIgnoreCase("USDC")) {
+            return "USD Coin (USDC)";
+        }
```

Comment:
This will be handled by #302 .  Can you remove this?  

Also it appears that there are many other currencies without names in the list.  

Other currencies from bitpay:
XRP, PAX, MRU, 

Some currencies appear to crash the app.  I will try to handle that in #302 also.

Reply:
Done
Crash issue resolved in https://github.com/dashevo/dash-wallet/pull/298/commits/afc6bd3719960f8810489950a47e0c3b0026e5ec#diff-d59db2bdbe7841e4cb222c288bc4e156R76